README.md

```diff
@@ -11,14 +11,23 @@ The old version (deprecated) is still available in the `legacy` branch.
 
 This is a very opinionated project, as I am a Tech Lead, this theme is very developer-focused.
 
+## Requirements
+
+### Nerd Fonts
+
+This theme requires the use of a patched font with Nerd Font. Ensure your terminal is set to use one before installing this theme. We recommend using FiraCode Nerd Font for this theme, but any patched font will do.
```

Comment:
We do not recommend any Nerd Font, because as long as the font is patched it will work. Currently I use San Francisco Mono patched and work fine. Maybe a link to Nerd font will be usefull

Reply:
This has been changed, please do review and add comments if you need something else for this hunk.